incubator/crunchy-postgresql-cluster/templates/master-pod.yaml

```diff
@@ -0,0 +1,61 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: "{{.Values.MasterName}}"
```

Comment:
I would suggest using a name that includes the release name to covert multiple installs in the same namespace